src/hotspot/share/prims/unsafe.cpp

```diff
@@ -108,7 +108,7 @@ static inline jlong field_offset_to_byte_offset(jlong field_offset) {
   return field_offset;
 }
 
-static inline jlong field_offset_from_byte_offset(jlong byte_offset) {
+static inline int field_offset_from_byte_offset(int byte_offset) {
```

Comment:
Nit: I wonder if the same fix is needed for the previous function at line 107 to make it consistent.
       But I guess there is no warning related to it.

Reply:
The names look almost the same, but field_offset_to_byte_offset() is called primarily passing jlong in several places so changing that would have had more fan out (and might not have been correct).
The callers of field_offset_from_byte_offset() had callers that passed int and expected int returns, so that's why I changed only that one.

Reply:
Okay, thanks.